test/http.go

```diff
@@ -29,6 +29,7 @@ type TestCase struct {
 	Headers         map[string]string `json:",omitempty"`
 	PathParams      map[string]string `json:",omitempty"`
 	FormParams      map[string]string `json:",omitempty"`
+	QueryParams     map[string]string `json:",omitempty"`
```

Comment:
Not used?

Reply:
not used here, but used in the dashboard PR

Reply:
Should it be a url.Values?